Mention other reviews? (#332)
greenelab/covid19-review: content/02.introduction.md

[Note: Maybe add a graph here, update as new reports come out.]
[So this is where manubot is particularly useful. it could pull from a public database / data table.]

> Some review articles on aspects of COVID-19 have already been published and may be of great usefulness.

Reviewer: ```suggestion
Some review articles on aspects of COVID-19 have already been published and may be very useful for contextualizing the disease within different disciplines.
```

> These have included reviews on epidemiological aspects (@doi:/10.1016/j.molmed.2020.02.008), immunological aspects (@doi:/10.1016/j.immuni.2020.05.002), and pharmacological treatments (@doi:/10.1016/j.immuni.2020.05.002).

Reviewer: ```suggestion
These have included reviews on the disease epidemiology [@doi:/10.1016/j.molmed.2020.02.008], immunological response [@doi:/10.1016/j.immuni.2020.05.002], and pharmacological treatments [@doi:/10.1016/j.immuni.2020.05.002].
```
The word "aspect" is used a few times, what do you think of this?

> However, research on these topics proceed so quickly that any static review is likely to quickly become dated.

Reviewer: ```suggestion
However, research on these topics is proceeding so quickly that any static review is likely to quickly become dated.
```
